subsys/bluetooth/host/hci_core.c

```diff
@@ -2399,8 +2399,8 @@ static void process_events(struct k_poll_event *ev, int count)
 #endif /* CONFIG_BT_ISO */
 #else
 #if defined(CONFIG_BT_ISO)
-/* command FIFO + MAX_ISO_CONN */
-#define EV_COUNT (1 + CONFIG_BT_ISO_MAX_CHAN)
+/* command FIFO + conn_change signal + MAX_ISO_CONN */
```

Comment:
I know this was a typo from my side, but could you please also update this? 
```suggestion
/* command FIFO + conn_change signal + MAX_ISO_CHAN */
```

Reply:
done